src/storage/db_interface_frontend.py

```diff
@@ -117,9 +117,9 @@ def _get_hid_firmware(self, uid):
         return None
 
     def _get_hid_fo(self, uid, root_uid):
-        file_object = self.file_objects.find_one({'_id': uid}, {'virtual_file_path': 1})
-        if file_object is not None:
-            return self._get_one_virtual_path_of_fo(file_object, root_uid)
+        fo_data = self.file_objects.find_one({'_id': uid}, {'virtual_file_path': 1})
```

Comment:
Not a fan of `fo_data` but it's okay

Reply:
I'm open to suggestions but it shouldn't be `file_object` since it isn't a `FileObject`